content/docs/reference/authorize-log-fields.mdx

```diff
@@ -275,7 +278,9 @@ authorize_log_fields:
   - request-id
   - path
   - ip
+```
 
+```yaml
```

Comment:
So, this one is configurable in the Console. The original ask here (#912) was to move examples into the Core tab if the setting only has Core examples, right? Again, I may have misunderstood the suggestion.

Reply:
Oh, I see. Yes, that was what I originally wrote in #912, but what do you think about moving all 'Examples' into their corresponding tabs? I think it would be good to keep the page structure consistent between reference pages.

Reply:
That's a good point, I'll have to comb through these again but it should be quick.